Extension/src/LanguageServer/client.ts

```diff
@@ -1046,8 +1051,12 @@ export class DefaultClient implements Client {
                     settings.defaultCompiler = compilerDefaults.compilerPath;
                     compilerDefaults = await this.requestCompiler(compilerPaths);
                     DefaultClient.updateClientConfigurations();
+                    action = "configure compiler";
+                    telemetry.logLanguageServerEvent('compilerNotification', { action });
```

Comment:
Can you pass the string instead of creating a new variable for it?

Reply:
Can you clarify? You can't pass the string as an argument; it has to be a {"key": value} pair.